proctor-common/src/main/java/com/indeed/proctor/common/ProctorResult.java

```diff
@@ -132,21 +135,54 @@ public ProctorResult(
      * @param buckets the resolved bucket for each test
      * @param allocations the determined allocation for each test
      * @param testDefinitions the original test definitions
+     * @param identifiers the identifiers used for determine groups
+     * @param inputContext the context variables
      */
+    @Deprecated
```

Comment:
why is this deprecated?

Reply:
I guess it is not really needed, I can remove.